docbook/Admin_Guide/en-US/config/issues.xml

```diff
@@ -0,0 +1,119 @@
+
+			</varlistentry>
+			 <varlistentry>
+				<term>$g_set_view_status_threshold</term>
+				 <listitem>
+					<para>
+						The threshold for a user to be able to set an issue to Private/Public.
+					</para>
+
+				</listitem>
+
+			</varlistentry>
+			 <varlistentry>
+				<term>$g_change_view_status_threshold</term>
+				 <listitem>
+					<para>
+						The threshold for a user to be able to update the view status while updating an issue.
+					</para>
+
+				</listitem>
+
+			</varlistentry>
+
+		</variablelist>
+
+	</section>
+	 <section id="admin.config.issues.limitreporters">
+		<title>"Limit reporters" configuration (deprecated)</title>
+		 <para>
+			When the option <literal>$g_limit_reporters</literal> is enabled, users that are reporters in a project, or lower access level, can only see the issues they reported. Any issue reporter by other users are not accessibel by them.
+		</para>
+		 <para>
+			The <literal>$g_limit_reporters</literal> option is supported only for ALL_PROJECTS, this means that it's a global setting that affects all projects.
+		</para>
+		 <para>
+			Note that the definition of <emphasis>reporter</emphasis> in this context is the actual access level for which a user is able to report issues, and is determined by <literal>$g_report_bug_threshold</literal>. Additionally, that threshold can have different values in each project. Being dependant of that threshold, the behaviour of this option is not well defined when the reporting threshold is configured as discrete values with gaps, instead of a simple threshold. In that scenario, the visibilty is determined upon the minimum access level contained in the <literal>$g_report_bug_threshold</literal> access levels array.
+		</para>
+		 <note>
+			<para>
+				The <literal>$g_limit_reporters</literal> option is deprecated in favour of <literal>$g_limit_view_unless_threshold</literal>. The new option will be available by default on new installations, or after disabling <literal>$g_limit_reporters</literal> if enabled in an existing instance.
+			</para>
+
+		</note>
+
+	</section>
+	 <section id="admin.config.issues.limitedview">
+		<title>Limited view configuration</title>
+		 <para>
+			The <literal>$g_limit_view_unless_threshold</literal> option allows to configure access limitations for users to view only those issues that they are involved with. Currently, this configuration supports the following conditions for a user to be able to view issues:
+			<itemizedlist>
+				<listitem>
+					<para>
+						The user is the reporter of the isssue.
+					</para>
+				</listitem>
+				 <listitem>
+					<para>
+						The user is handling the issue.
+					</para>
+				</listitem>
+				 <listitem>
+					<para>
+						The user is monitoring the issue
+					</para>
+				</listitem>
+
+			</itemizedlist>
+
+		</para>
+		 <para>
+			This configuration option can be set individually for each project. The dafult value is ANYBODY, so this limitation is effectively disabled by default.
```

Comment:
<del>The dafult value is ANYBODY, so this limitation is effectively disabled by default.</del> 
--> `It defaults to ANYBODY, effectively disabling the limitation (i.e. users can see all issues).`